### PR TITLE
Data mod

### PR DESCRIPTION
**Pull request by EldritchJS**
Modified data handler to accommodate three operating modes based on the value of OP_MODE env variable. Single mode takes either the first field (or just the entry if no separator) as its input message, while dual takes the full input as its value to be counted towards top-k. linear does the same as dual (included for consistency with data publisher)

### CODE REVIEW

**Comment by erikerlandson** [comment]
so the "default" is first value in "v1,v2,v3...", otherwise just the whole field?

**Comment by EldritchJS** [comment]
Default is the first field in the message, otherwise just the entire message

**Comment by erikerlandson** [comment]
LGTM

**Comment by erikerlandson** [comment]
I may be confused by intended semantics of "linear" - in the original description on this PR it says "same as single mode" but now it is same as dual.

**Comment by EldritchJS** [comment]
Good catch, single takes only the first field, linear and double take the entire message as is.

File: src/main/scala/io/radanalytics/equoid/DataHandler.scala [hunk 1]
```diff
@@ -23,12 +22,18 @@ object DataHandler {
...
   }
 
-  def messageConverter(message: Message): Option[String] = {
+  def messageConverter(message: Message, opMode: String): Option[String] = {
 
     message.getBody match {
       case body: AmqpValue => {
         val itemID: String = body.getValue.asInstanceOf[String]
-        Some(itemID)
+        //val primaryVal: String = itemID.split(",")(0)
```
Comment (erikerlandson) [inline src/main/scala/io/radanalytics/equoid/DataHandler.scala]: nit: if this is working, delete this comment

**Comment by erikerlandson** [comment]
LGTM; we can merge this and address any follow-up on a future PR

**Pull request merged**